src/fides/data/sample_project/docker-compose.yml

```diff
@@ -42,6 +42,8 @@ services:
       FIDES_DEPLOY__CONNECTORS__MONGO_DEFAULTAUTHDB: mongo_test
       FIDES_DEPLOY__CONNECTORS__MONGO_USERNAME: mongo_user
       FIDES_DEPLOY__CONNECTORS__MONGO_PASSWORD: mongo_pass
+      # uncomment the line below to enable the "loyalty database"
+      # FIDES_DEPLOY__CONNECTORS__POSTGRES_LOYALTY__HOST: postgres-test
```

Comment:
NOTE: locally, the `fides_env` nox command will automatically provide your local `.env` file to `fides deploy`, so you shouldn't need to even modify this compose file - just have `FIDES_DEPLOY__CONNECTORS__POSTGRES_LOYALTY__HOST` set in your .env and it's good to go.

Reply:
Nice! In that case, I'll remove this mapping from the docker-compose